dashboard/config/scripts_json/csd1-2021.script_json

```diff
@@ -7142,5 +7142,8 @@
         "objective.key": "1e8b24b5-f8b6-46a6-b0b1-aba1633a02f0"
       }
     }
+  ],
+  "lessons_standards": [
+
```

Comment:
you can probably save yourself some merge headaches later by reverting files containing no programming expressions. I went one step further and removed all lines changed that were not specifically the part i cared about adding. I can show you some git tricks for this. the goal would be to avoid painful merge conflicts on levelbuilder. if unsure how much to invest here, at least reverting changes to `serialized_at` could be a good starting point.